### ISSUE TITLE
Neuralnet Training Recommendations

### ISSUE THREAD

**Issue by kastnerkyle** [kastnerkyle/todo#52]
via Ian Goodfellow
Many machines, random search.

Otherwise, set the momentum to start at 0.5, increase over time to something like 0.6-0.9, set the learning rate to decay 100X or so over the course of a few hundred epochs, and then do a binary search for the best initial learning rate.
